postgres/managing/forking.html.markerb

```diff
@@ -0,0 +1,46 @@
+---
+title: "Fork a volume from a Postgres app"
+objective: Create a new Postgres app with the data from an existing Postgres app.
+layout: framework_docs
+order: 30
+---
+
+<%= partial "docs/postgres/partials/for-machines" %>
+
+A volume fork is a new volume that contains an exact copy of the data from a source volume. For Postgres apps, you use the `--fork-from` flag to specify a source Postgres app when you create a new Postgres app. The new Postgres app has a new volume that contains a block-level snapshot of the data from the source app's volume.
+
+Use volume forking to create identical staging or test environments, troubleshoot issues, analyze data, or test database performance without impacting your production database. You can also use volume forks for PR review environments. A PR review environment is especially useful when a PR contains database migrations that you don’t want to run against your production database.
+
+<div class="important icon">
+**Important:** The new volume and the source volume are not synced. The new volume is in the same region as the source app's volume, but on a different physical host. Forking directly to a new region isn't supported yet.
```

Comment:
This might cause some confusion, as noted in the other PR.  The new volume is synced to a point-in-time, but will not continue to sync beyond that.